main.go

```diff
@@ -63,6 +63,7 @@ func mainWithError() error {
 		c := app.MutatorConfig{
 			K8sClient: cfg.K8sClient,
 			Logger:    newLogger,
+			Provider:  cfg.Provider,
```

Comment:
Where does the provider actually come from?

Reply:
Nvm, found it

Reply:
It's a required command-line parameter. Defined in the [config repo](https://github.com/giantswarm/config/blob/main/default/apps/app-admission-controller/configmap-values.yaml.template#L4-L5), passed on in [Deployment's `args`](https://github.com/giantswarm/app-admission-controller/blob/master/helm/app-admission-controller/templates/deployment.yaml#L37-L41).